src/assets/js/irma-functions.js

```diff
@@ -86,12 +86,12 @@ function irmaDoSession(request, header = '') {
     return irmaPopup.start()
 }
 
-const irmaDisclosedResultSingleRawValueFromIndex = function (result, index) {
-  return result.disclosed[index][0].rawvalue || '';
+const irmaDisclosedResultSingleRawValueFromIndex = function (result, credential_index=0, attribute_index=0) {
+  return result.disclosed[credential_index][attribute_index].rawvalue || '';
```

Comment:
Just to follow JS conventions
```suggestion
const irmaDisclosedResultSingleRawValueFromIndex = function (result, credentialIndex = 0, attributeIndex = 0) {
  return result.disclosed[credentialIndex][attributeIndex].rawvalue || '';
```